ops/utils.go

```diff
@@ -78,6 +78,22 @@ func OffsetTensorIfNegative(t tensor.Tensor, offset int) error {
 	return nil
 }
 
+// AnyToInt casts the given data to an int, but only if the data is of some sort of int type.
+func AnyToInt(value interface{}) (int, error) {
```

Comment:
This can be done using generics? Then the type assert is not necessary and you don't need to return  an error since the compiler will check if the conversion is possible

Reply:
I don't think this is possible because the usage is in CumSum Apply with the result of `ScalarValue()`, which is an interface. The type of this is determined by the typing system of `gorgonia.Tensor`